systems/plants/RigidBody.cpp

```diff
@@ -62,7 +64,7 @@ void RigidBody::setN(int nq, int nv) {
 }
 
 
-void RigidBody::setJoint(std::unique_ptr<DrakeJoint> new_joint)
+void RigidBody::setJoint(std::unique_ptr<DrakeJoint>& new_joint)
```

Comment:
I believe the conventional wisdom is to take `unique_ptr` arguments by value if you're going to move from them. See 3.c in [this article](http://herbsutter.com/2013/06/05/gotw-91-solution-smart-pointer-parameters/) of Herb Sutter's.